docs/modeling/models.rst

```diff
@@ -1,9 +1,23 @@
 .. include:: links.inc
 .. _modeling-instantiating:
 
-***********************************
-Instantiating and Evaluating Models
-***********************************
+************
+Using Models
+************
+
+The module currently supports 1-D and 2-D models. It is designed to be
```

Comment:
I'd strike that first sentence altogether.  This module actually supports N-dimensional models (including zero-dimensional although I don't know what use that is).